### DISCUSSION TITLE
Backup/Restore Wiki JS between two servers

### DISCUSSION THREAD

**Discussion by karimmahgoub** [requarks/wiki#6041] [Help / Questions] [answered]
Hello All,
I want to move Wiki JS installation from K8s environment to docker containers environment.
I have used the GitHub option in Storage module for backup and restore and I can see all the pages moved to the new server, but users and groups weren't moved. I am missing something or the Github option doesn't include users/groups?
Also I have tried the option of making a backup of the database but I can't get the its password.
Also I have tried the export option in "utilities" but not sure how to import the output in the new installation.
Thanks.

**Comment by NGPixel** [comment, marked as answer]
You need to copy the DB in order to keep users/groups. Storage options are strictly for pages / assets.

**Reply by karimmahgoub** [reply]
Thanks for the fast reply. So there is no way to use the json files of users and groups in the new server?

**Reply by NGPixel** [reply]
Not at the moment no.

**Reply by karimmahgoub** [reply]
Thanks for the info. I will try to find a way to reset the password or move the volume and see how I will deal with it.
